packages/slice-machine/src/features/customTypes/customTypesBuilder/sliceCreationOptions.tsx

```diff
@@ -0,0 +1,80 @@
+import { BackgroundIcon } from "@prismicio/editor-ui";
+
+import { useSectionsNamingExperimentReturnType } from "@/features/builder/useSectionsNamingExperiment";
+import { capitalizeFirstLetter, pluralize } from "@/utils/textConversion";
+
+type SliceCreationOptionArgs = {
+  menuType: "ActionList" | "Dropdown";
+  sectionsNamingExperiment: useSectionsNamingExperimentReturnType;
+};
+
+export const getSliceCreationOptions = (args: SliceCreationOptionArgs) => {
+  const { menuType, sectionsNamingExperiment } = args;
+
+  return {
+    fromImage: {
+      BackgroundIcon: (
+        <BackgroundIcon
+          name="autoFixHigh"
+          size={menuType === "ActionList" ? "small" : "extraSmall"}
+          iconSize={menuType === "ActionList" ? "medium" : "small"}
+          color="purple"
+          variant="solid"
+          radius={6}
+        />
+      ),
+      title: "Generate from image",
+      description: `Build a ${capitalizeFirstLetter(
+        sectionsNamingExperiment.value,
+      )} based on your design image.`,
```

Comment:
I thought we said that these were not going to be capitalized, right?

what we want is _Build a section based on your design image._ and not _Build a Section based on your design image._